package.json

```diff
@@ -24,6 +25,8 @@
   },
   "homepage": "https://github.com/c-hive/team-contribution-calendar#readme",
   "devDependencies": {
+    "parcel-bundler": "^1.12.3",
+    "parcel-plugin-inlinesvg": "^0.1.1",
```

Comment:
Interestingly enough, it seems as if `parcel-plugin-inlinesvg` [doesn't support](https://github.com/albinotonnina/parcel-plugin-inlinesvg/issues/9#issuecomment-412360600) `parcel-bundler` over 1.9.0+. But, since it appears to be working, let's deem it an additional information.